web-bundle/src/main/resources/com/graphhopper/maps/js/main-template.js

```diff
@@ -78,17 +78,21 @@ $(document).ready(function (e) {
     // todo: the idea was to highlight everything so if we start typing the example is overwritten. But unfortunately
     // this does not work. And not even sure this is so useful?
     showCustomModelExample();
-    cmEditor.validListener = function(valid) {
+    cmEditor.validListener = function (valid) {
         $("#custom-model-search-button").prop('disabled', !valid);
     };
-    $("#custom-model-button").click(function() {
+    ghRequest.cmEditor = cmEditor;
+    ghRequest.cmEditorActive = false;
+    $("#custom-model-button").click(function () {
         $("#custom-model-box").toggle();
+        ghRequest.cmEditorActive = !ghRequest.cmEditorActive;
         // avoid default action, so use a different search button
         $("#searchButton").toggle();
         mapLayer.adjustMapSize();
         cmEditor.cm.refresh();
         cmEditor.cm.focus();
-        cmEditor.cm.setCursor(cmEditor.cm.lineCount())
+        cmEditor.cm.setCursor(cmEditor.cm.lineCount());
+        sendCustomData();
```

Comment:
I thought it is more intuitive to update the route whenever we open/close the custom model box